app/controllers/api/v1/users_controller.rb

```diff
@@ -3,14 +3,18 @@ module V1
     class UsersController < Api::V1::ApiController
       before_action :auth_user
 
-      def show; end
+      def show
+        @user = UserPresenter.new(current_user)
```

Comment:
what about moving the `UserPresenter.new(current_user)` to a private method as explained in this post: https://blog.rstankov.com/rails-anti-pattern-setting-view-variables-in-before-actions/ ?
Just to be a bit more DRY